feat(apple): Swift concurrency support (#7066)
getsentry/sentry-docs: src/platform-includes/getting-started-config/apple.mdx

}];
```

> ## Have your Swift concurrency stack traces stitched together

Reviewer: ```suggestion
## Stitch Together Swift Concurrency Stack Traces
```

> > This feature is experimental and may have bugs.

Reviewer: ```suggestion
<Note>
This feature is experimental and may have bugs.
</Note>
```

Reviewer: Let's use note formatting for this

> If you are using Swift concurrency, you can have your stack traces stitched together. This means that you can see the full stack trace of your async code.

Reviewer: ```suggestion
If you use Swift concurrency, this feature will stitch your stack traces together. That means you will be able to see the full stack trace of your async code.
```

For this to happen you need to enable the `swiftAsyncStacktraces` option.

> You can also enable this in your objc project, however, only async code written in Swift will be stitched together.

Reviewer: ```suggestion
You can also enable this in your Objective-C project, however, only async code written in Swift will be stitched together.
```
